hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/StorageBasedLockConfig.java

```diff
@@ -83,13 +83,13 @@ private void checkRequiredProps() {
         throw new IllegalArgumentException(BASE_PATH.key() + notExistsMsg);
       }
       if (lockConfig.getLongOrDefault(VALIDITY_TIMEOUT_SECONDS) < lockConfig.getLongOrDefault(HEARTBEAT_POLL_SECONDS)
-          * 3) {
+          * 10) {
```

Comment:
if you get any more feedback from danny or someone before we land this patch, can you fix L 35 var name to LOCK_VALIDITY_TIMEOUT_SECS

Reply:
Actually @yihua suggested `VALIDITY_TIMEOUT_SECONDS` in an earlier PR it used to be `LOCK_VALIDITY_TIMEOUT_MS`